packages/preset-typescript/README.md

```diff
@@ -1,6 +1,6 @@
-# Typescript preset for Storybook
+# TypeScript preset for Storybook
```

Comment:
I know this is correct, but it makes my eyes bleed 😭

Reply:
Hahaha, I'm a little OCD.